62.CAD/common.hlsl

```diff
@@ -53,11 +98,21 @@ struct Globals
 
 struct LineStyle
 {
+    // common data
     float4 color;
     float screenSpaceLineWidth;
     float worldSpaceLineWidth;
-    // TODO[Przemek]: Anything info related to the stipple pattern will be here
-    float _pad[2u];
+    
+    // stipple pattern data
+    static const uint32_t STIPPLE_PATTERN_MAX_SZ = 15u;
+    int32_t stipplePatternSize;
+    float recpiprocalStipplePatternLen;
+    float stipplePattern[STIPPLE_PATTERN_MAX_SZ-1];
+    float phaseShift;
+    float _pad[1u];
```

Comment:
you can add one to the stipple pattern size if it's unused :D

Reply:
done